src/resources/postcss/components/your-tickets/_attendees.pcss

```diff
@@ -0,0 +1,34 @@
+.tec-tickets__attendees-list-order-attendees {
+	.event-tickets-wallet-plus__your_tickets_title_area {
+		padding: 12px 0;
+	}
+
+	.tec-tickets__attendees-list-attendee-row {
+		align-items: center;
+		border-left: 1px solid var(--tec-color-border-secondary);
+		border-right: 1px solid var(--tec-color-border-secondary);
+		border-top: 1px solid var(--tec-color-border-secondary);
+		display: flex;
+		margin: 0;
+		padding: 20px;
```

Comment:
We have a [spacer variable](https://github.com/the-events-calendar/tribe-common-styles/blob/main/variables/_spacers.pcss#L10) that's 20px.